src/main/resources/templates/mail/welcome.html

```diff
@@ -1,213 +1,102 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HIPAA Consent</title>
+
+  <style>
+    *{
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+      font-family: sans-serif;
     }
 
-    @font-face {
-      font-family: 'Source Sans Pro';
-      font-style: normal;
-      font-weight: 700;
-      src: local('Source Sans Pro Bold'), local('SourceSansPro-Bold'), url(https://fonts.gstatic.com/s/sourcesanspro/v10/toadOcfmlt9b38dHJxOBGFkQc6VGVFSmCnC_l7QZG60.woff) format('woff');
+    body{
+      height: 100%;
+      display:grid;
+      justify-content: center;
+      align-items: center;
+      padding: 10px 10px 50px;
+      background: white;
     }
-  }
-
-  body,
-  table,
-  td,
-  a {
-    -ms-text-size-adjust: 100%; /* 1 */
-    -webkit-text-size-adjust: 100%; /* 2 */
-  }
-
-  table,
-  td {
-    mso-table-rspace: 0pt;
-    mso-table-lspace: 0pt;
-  }
-
-  img {
-    -ms-interpolation-mode: bicubic;
-  }
-
-  a[x-apple-data-detectors] {
-    font-family: inherit !important;
-    font-size: inherit !important;
-    font-weight: inherit !important;
-    line-height: inherit !important;
-    color: inherit !important;
-    text-decoration: none !important;
-  }
-
 
-  div[style*="margin: 16px 0;"] {
-    margin: 0 !important;
-  }
-
-  body {
-    width: 100% !important;
-    height: 100% !important;
-    padding: 0 !important;
-    margin: 0 !important;
-  }
+    .logo{
+      display: block;
+      margin-left: auto;
+      margin-right: auto;
+      margin-top: auto;
+    }
 
+    h1 {
+      text-align: center;
+    }
 
-  table {
-    border-collapse: collapse !important;
-  }
+    h2 {
+      text-align: center;
+    }
 
-  a {
-    color: black;
-  }
+    .container{
+      max-width: 800px;
+      width: 100%;
+      background-color: #fff;
+      padding: 25px 30px;
+      border-radius: 10px;
+      box-shadow: 0 0 5px #fcc629;
+    }
 
-  img {
-    height: auto;
-    line-height: 100%;
-    text-decoration: none;
-    border: 0;
-    outline: none;
-  }
+    .buttons:hover{
+      background: #ffc629;
+    }
 
+    .buttons{
+      height: 45px;
+      width: 49%;
+      background-color: #a7a7a7;
+      border-radius: 5px;
+      border: none;
+      margin-left: auto;
+      margin-right: auto;
 
+    }
+    .welcome{
+      max-width: 800px;
+      width: 100%;
+      background-color: #fff;
+      padding: 25px 30px;
+      border-radius: 10px;
+      box-shadow: 0 0 5px #fcc629;
+    }
+   .verify{
+     text-align: center;
+   }
   </style>
-
 </head>
-<body bgcolor="#e9ecef">
 
-  <!-- start preheader -->
-  <div class="preheader" style="display: none; max-width: 0; max-height: 0; overflow: hidden; font-size: 1px; line-height: 1px; color: #fff; opacity: 0;">
-    A preheader is the short summary text that follows the subject line when an email is viewed in the inbox.
+<body>
+<img class="logo" src="/img/logo.png" alt="logo"><br>

```

Comment:
This can be fixed in another PR if you want, but email clients have no context about our website root URL, so all URLs have to be absolute. Changing this to `https://virtudoc.herokuapp.com/img/logo.png` should make the logo appear inside SMTP4dev and in email clients. (Unlike the button link, this should work locally, but this is also another good example of where that URLHelper I mentioned might help).